ga4gh/converters.py

```diff
@@ -0,0 +1,42 @@
+"""
+Provides classes that convert protocol requests to genomics data files
```

Comment:
It's not clear to me what it means to convert a request (in)to a file? Is this just the wrong preposition (or one of the transatlantic preposition shift issue), or do we need to be more explicit?